network-api/networkapi/wagtailpages/pagemodels/libraries/rcc/landing_page.py

```diff
@@ -1,7 +1,7 @@
 from django.apps import apps
 from django.db import models
 from wagtail import models as wagtail_models
-from wagtail.admin.panels import FieldPanel
+from wagtail.admin.panels import FieldPanel, InlinePanel
```

Comment:
**Change**

Some of the imports that are being introduced aren't matching the current module level style.
I think this should be"

```python
from wagtail.admin import panels

...
panels.FieldPanel,
panels.InlinePanel
...
```